lib/asm-parser.js

```diff
@@ -300,6 +300,10 @@ export class AsmParser extends AsmRegex {
                         file: !file.match(stdInLooking) ? file : null,
                         line: sourceLine,
                     };
+                    const sourceCol = parseInt(match[3]);
+                    if (!isNaN(sourceCol)) {
+                        source.column = sourceCol;
```

Comment:
I feel like maybe this should also not be set when sourceCol is 0. That should be clear enough that it's about the entire line and/or that the asm doesn't contain the info.

Reply:
Makes sense. I'll skip 0s too.